spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/SparkAtlasEventTracker.scala

```diff
@@ -115,4 +115,4 @@ class SparkAtlasEventTracker(atlasClient: AtlasClient, atlasClientConf: AtlasCli
         false
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Let's revert this change on this file, too.

Reply:
error file=/Users/mtang/Dev/spark/spark-atlas/spark-atlas-connector/spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/SparkAtlasEventTracker.scala message=File must end with newline character